platform/features/imagery/src/controllers/ImageryController.js

```diff
@@ -111,6 +111,10 @@ define(
                             this.updateHistory(datum);
                         }, this);
                         this.requestLad(true);
```

Comment:
I think that the problem is actually here. requestLAD = requestLatestAvailableData. In the implementation of `requestLAD` it takes the first image, rather than the last one.

Reply:
@akhenry - The issue of latest image not updating was only prevalent when no new data was coming in. I think requesting an update of current value when history is updated would fix this issue. Let me know if you still think my approach is not correct.

Reply:
@dtailor90 The [request strategy docs](https://github.com/nasa/openmct/blob/master/API.md#request-strategies-draft) are in a draft state, but in essence, request strategies may not be implemented by a telemetry provider.   So even thought you are making a LAD request, you may not get a LAD response of 1-- it may contain more than one value.  So you need to display the last datum in the response array i.e. `values[values.length - 1]`.

Reply:
@larkin, ok that makes sense. I was assuming the response was always an array with one value.

Reply:
What @larkin said. My apologies, my comment assumed knowledge about request strategies that I should not have assumed.